src/Resources/laravel-debugbar.css

```diff
@@ -226,10 +226,6 @@ div.phpdebugbar-header {
-}
 
```

Comment:
Could this change be unintended? I think it removes the Laravel logo:

<img width="152" alt="Screenshot 2024-04-03 at 10 03 57" src="https://github.com/barryvdh/laravel-debugbar/assets/18613261/793f3594-d77f-4989-9227-323a19cb043f">

Reply:
Whoops